challenge.rb

```diff
@@ -57,8 +68,22 @@ def ac_monitor current_temp, ac_working, desired_temp
 # - return "fizzbuzz" if `num` is evenly divisible by both 3 and 5
 # - return the value of `num` if `num` isn't divisible by either 3 or 5
 
-# Replace this comment with your code.
-
+def fizz_buzz_calculator num
+
+  i = 1
+  while i <= num
+    if num % 3 == 0 && num % 5 == 0
+      return 'fizzbuzz'
+    elsif num % 3 == 0
+      return 'fizz'
+    elsif num % 5 == 0
+      return 'buzz'
+    else
+      return num
+    end
+    i +=1
+  end
+end
```

Comment:
Not quite - this method was only supposed to figure out the result, not loop through all the numbers (also, the `return` will cause your loop to stop cold after the first run through). 

Since your logic was correct, though, I'll give you partial credit on this one.